confirm-deposit.md

```diff
@@ -0,0 +1,120 @@
+# Confirm Deposits to the Exchange
+
+Deposits are made in several stages. 
+After the user deposits funds to their assigned reception wallet (step 1 in [Detecting Deposits](https://docs.flare.network/exchange/architecture/#detecting-deposits)), the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange) monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a [blockchain](https://docs.flare.network/tech/glossary/), multiple [validators](https://docs.flare.network/tech/validators/) examine newly created transactions and decide on whether they are valid and which [block](https://docs.flare.network/tech/glossary/) to add to the chain next. 
+It takes time for all the validators to agree. 
+Until they reach agreement, validators may add or revert transactions to a block. 
+After the validation process is complete, reversing a transaction is very unlikely. 
+To avoid the risk of confirming a block that could be reverted, confirm deposits several blocks back (for example, five blocks back as in the code below). 
+Older blocks are less likely to be reverted.
+
+Newly submitted deposits to the receiving address are labeled "pending."
+After five more blocks have been created, valid deposits are labeled "confirmed."
+
+Run the code below to see pending and confirmed transactions and which block number they were added to.
+
+## One-Time Setup
+
+1. Install [node.js](https://nodejs.org/en/download/). 
+
+   On the command line, run `node -v`  to check your installation.
+
+2. Clone the Flare repository: 
+
+   ```
+   git clone https://github.com/flare-foundation/flare.git
+   cd flare
+   ```
+
+   See [the Flare repository on GitHub](https://github.com/flare-foundation/flare).
+
+3. Install the web3.js library: 
+
+   `npm install web3`
+
+   Run `npm web3 -v`  to check your installation.
+
+   See [Adding web3.js](https://web3js.readthedocs.io).
+
+4. Build Flare with this script:
+
+   `./scripts/build.sh`
+
+## Run the Code
+
+The following code:
+
+* Sets up the deposit by instantiating `web3`, assigning the endpoint URL, and assigning the `receivingAddress`
```

Comment:
Missing period.

Reply:
I went back and forth about that. Sometimes I leave a period out after code, so it doesn't get confused with the code. In this case it's really clear, so agreed.

Reply:
I went back and forth about that. Sometimes I leave a period out after code, so it doesn't get confused with the code, but it's really clear here, so agreed.